src/components/Restoration/RestorationType.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div id="restoration-type">
+  <div
+    id="restoration-type"
+    :class="{ 'invalid-section': invalidSection }"
```

Comment:
This was previously in the parent component, but to make this consistent with the shared ApprovalType component, I moved it here.